config/kubernetes/org.yaml

```diff
@@ -1863,7 +1863,7 @@ teams:
     - gmarek
     - wojtek-t
     privacy: closed
-  product-security-team:
+  product-security-committee:
```

Comment:
@cblecker @fejta peribolos should delete the old team and create a new team for PSC, right? Can you sanity check this once?

Reply:
Explicitly holding for this comment.

/hold

Reply:
If you do something like this with the previous name, it will rename the team, instead of delete/recreate:
https://github.com/kubernetes/org/blob/9794f30916f026dc3b03bb7af78f7ec2bea90bb5/config/kubernetes/sig-pm/teams.yaml#L10-L11

Reply:
Ah right, I was pretty sure I was forgetting something. :)

Done. :+1: